Task-based edit: Managing Custom Data (#6185)
braze-inc/braze-docs: _docs/_user_guide/data_and_analytics/custom_data/managing_custom_data.md

## Blocklisting custom data

You may occasionally identify custom attributes, custom events, or purchase events that either consume too many data points, are no longer useful to your marketing strategy, or were recorded in error. To stop this data from being sent to Braze, you can blocklist a custom data object while your engineering team works to remove it from the backend of your app or website.

Blocklisting prevents a particular custom data object from being recorded by Braze going forward. Blocklisting does not remove data from user profiles or retroactively decrease the amount of data points incurred for that custom data object. 

For instructions and details about blocklisting custom data, refer to [Custom event and attribute management]({{site.baseurl}}/user_guide/administrative/app_settings/manage_app_group/custom_event_and_attribute_management/#blocklisting-custom-attributes-custom-events-and-products).

### Deleting custom data

> After you blocklist the custom data object and remove references to it from your app or website, the custom data can be deleted. For help deleting custom data, please reach out to your customer success manager or the Braze support team.

Reviewer: ```suggestion
The custom data can be deleted after you blocklist the custom data object and remove references from your app or website. For help deleting custom data, contact your customer success manager or the Support team.
```
